dex-it-common/src/main/scala/com/wavesplatform/dex/it/api/HasToxiProxy.scala

```diff
@@ -27,7 +27,7 @@ trait HasToxiProxy { self: BaseContainersKit =>
 
   protected def getInnerToxiProxyPort(proxy: ContainerProxy): Int =
     toxiContainer.getContainerInfo.getNetworkSettings.getPorts.getBindings.asScala
-      .find { case (_, bindings) => bindings.head.getHostPortSpec == proxy.proxyPort.toString }
+      .find { case (_, bindings) => Option(bindings).flatMap(_.headOption).exists(_.getHostPortSpec == proxy.proxyPort.toString) }
```

Comment:
New version of testcontainers returns array bindings as `Null` if it was dynamically set by docker. It caused NPE at container's start, because we log every binding at start. So, I wrapped bindings in Option everywhere I found it.